Geometry_Engine/Query/IsClockwise.cs

```diff
@@ -115,16 +121,38 @@ public static bool IsClockwise(this Arc arc, Vector axis, double tolerance = Tol
             return ((normal.DotProduct(axis) < 0) != (arc.Angle() > Math.PI));       
         }
 
+        /***************************************************/
+
+        public static bool IsClockwise(this Circle curve, Vector axis, double tolerance = Tolerance.Distance)
```

Comment:
Why not base the IsClockwise between the sign of dot product of circle normal and reference vector?

Reply:
Good idea 👍